lib/ansible/modules/package.py

```diff
@@ -83,4 +83,10 @@
       - httpd
       - mariadb-server
     state: latest
+
+- name: Use YUM to Install a Package
+  ansible.builtin.package:
+    name: httpd
+    state: present
+    use: yum    
```

Comment:
We should probably use `dnf` here instead as the `yum` module is being removed in the next release.

Reply:
@mkrizek 

Updated the changes recommended